jest.d.ts

```diff
@@ -7,7 +7,6 @@ declare namespace jest {
     toRender(component: React.ComponentType): R;
     toHaveState(selector: string, expected: unknown): R;
     // The following matchers are part of https://www.npmjs.com/package/jest-dom
-    toBeInTheDocument: () => R;
```

Comment:
Now comes for free as far as I can tell.

<img width="1116" alt="CleanShot 2020-01-22 at 18 17 39@2x" src="https://user-images.githubusercontent.com/1877073/72917708-9ed43800-3d44-11ea-9bcc-347c648eca08.png">

Reply:
Ah, good to know!